addons/web/static/src/views/list/list_renderer.js

```diff
@@ -1370,30 +1370,6 @@ export class ListRenderer extends Component {
         return false;
     }
 
-    /**
```

Comment:
[REF] web: focus of a X2many

This commit changes the behaviour of a X2many in a
form view when a "tab" or "shift-tab" make it focused.

Before this commit, a new empty row would be created automatically.

After this commit, the button "Add a line" takes focus and no new line is created.
If one wants to add one, that can be done by pressing "Enter".